profiles/models.py

```diff
@@ -0,0 +1,48 @@
+"""
+Models for user profile
+"""
+
+from django.contrib.auth.models import User
+from django.db import models
+from jsonfield import JSONField
+
+
+class Profile(models.Model):
+    """
+    A user profile
+    """
+    PRIVATE = 'private'
+    PUBLIC_TO_MM = 'public_to_mm'
+    PUBLIC = 'public'
+    ACCOUNT_PRIVACY_CHOICES = (
+        (PRIVATE, 'Private'),
+        (PUBLIC_TO_MM, 'Public to logged in users'),
+        (PUBLIC, 'Public to everyone'),
+    )
+
+    user = models.ForeignKey(User)
+    account_privacy = models.TextField(
+        default=PRIVATE,
+        choices=ACCOUNT_PRIVACY_CHOICES,
+    )
+    email_optin = models.BooleanField(default=False)
+    employer = models.TextField(blank=True, null=True)
+    job_title = models.TextField(blank=True, null=True)
+    state_or_territory = models.TextField(blank=True, null=True)
+
+    name = models.TextField(blank=True, null=True)
+    bio = models.TextField(blank=True, null=True)
+    country = models.TextField(blank=True, null=True)
+    has_profile_image = models.BooleanField()
+    profile_url_full = models.TextField(blank=True, null=True)
+    profile_url_large = models.TextField(blank=True, null=True)
+    profile_url_medium = models.TextField(blank=True, null=True)
+    profile_url_small = models.TextField(blank=True, null=True)
```

Comment:
I'm a bit reluctant to use UrlField or anything else here since these fields are copied from edX (at least initially) so they're not guaranteed to validate